pkg/cmd/experimental/router/router.go

```diff
@@ -191,23 +227,9 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out io.Writer)
 								Template: &kapi.PodTemplateSpec{
 									ObjectMeta: kapi.ObjectMeta{Labels: label},
 									Spec: kapi.PodSpec{
-										Containers: []kapi.Container{
-											{
-												Name:  "router",
-												Image: image,
-												Ports: ports,
-												Env:   env.List(),
-												LivenessProbe: &kapi.Probe{
-													Handler: kapi.Handler{
-														TCPSocket: &kapi.TCPSocketAction{
-															Port: kutil.IntOrString{
-																IntVal: ports[0].ContainerPort,
-															},
-														},
-													},
-												},
-											},
-										},
+										Containers:  getRouterPodContainers(cfg, env),
+										Volumes:     getFailoverContainerVolumes(),
+										HostNetwork: true,
```

Comment:
I still have an issue here with this configuration. For some reason, the containers that get started are not started using the HostNetwork. 

However my test json script works fine.   See:  https://gist.github.com/ramr/b6389a7bfbade8df8cdc   for details.
That one works with ```osc create -f  foo.json``` 

So something in this API interaction is causing issues. Spent a good day plus trying to figure this out - thought originally it was an issue w/  api version v1beta3 to v1beta1 conversions but the above mentioned json config script works fine. So its something in here that am not doing correct.  Any help would be greatly appreciated. thx